tests/suite/P130.ok

```diff
@@ -10,26 +10,26 @@ RND FUNCTION.
              BEGIN TEST
 
 POSITION      VALUE
- 1             .840188 
- 2             .394383 
- 3             .783099 
- 4             .79844 
- 5             .911647 
- 6             .197551 
- 7             .335223 
- 8             .76823 
- 9             .277775 
- 10            .55397 
- 11            .477397 
- 12            .628871 
- 13            .364784 
- 14            .513401 
- 15            .95223 
- 16            .916195 
- 17            .635712 
- 18            .717297 
- 19            .141603 
- 20            .606969 
+ 1             .168183 
```

Comment:
This is odd. Did we upload the wrong test program?

Reply:
No, but the numbers are random and there is no spec what is the seed and how exactly to generate the numbers (luckely enough).